openvmm/openvmm_entry/src/cli_args.rs

```diff
@@ -637,6 +637,11 @@ pub enum DiskCliKind {
         create: bool,
         disk: Box<DiskCliKind>,
     },
+    // autocache:[key]:<kind>
+    AutoCacheSqlite {
+        key: String,
```

Comment:
lets make this an Option here, instead of pushing that `is_empty()` logic down the chain